addons/im_livechat/static/src/core/web/discuss_sidebar_category_patch.xml

```diff
@@ -14,65 +14,61 @@
+        <span t-if="livechatThread.channel_type === 'livechat' and (['waiting', 'need_help'].includes(livechatThread.livechat_status) or livechatThread.livechat_end_dt)" class="o-livechat-LivechatStatusLabel" t-att-title="livechatThread.livechatStatusLabel" t-att-class="{
             'o-xsmaller': !env.inLivechatInfoPanel,
-            'position-absolute top-0 start-0 z-1': env.inDiscussSidebar or env.inChatBubble,
-            'm-n1': env.inDiscussSidebar,
-            'm-n2': env.inChatBubble,
-            'o-warning': livechatThread.livechat_status === 'waiting',
-            'o-info': livechatThread.livechat_status === 'need_help',
+            'o-livechat-LivechatStatusLabel-Sidebar position-absolute top-0 start-0': env.inDiscussSidebar or env.inChatBubble,
+            'end-0 bottom-0 rounded-2 border': env.inDiscussSidebar,
+            'm-n2 z-1': env.inChatBubble,
+            'text-warning': livechatThread.livechat_status === 'waiting',
+            'text-info': livechatThread.livechat_status === 'need_help',
+            'bg-warning border-warning': livechatThread.livechat_status === 'waiting' and env.inDiscussSidebar,
+            'bg-info border-info': livechatThread.livechat_status === 'need_help' and env.inDiscussSidebar,
         }">
-            <i class="fa fa-circle o-livechat-LivechatStatusLabel-bubble fa-stack-1x" t-att-class="{
-                'o-warning': livechatThread.livechat_status === 'waiting',
-                'o-info': livechatThread.livechat_status === 'need_help',
-                'o-secondary o-white': livechatThread.livechat_end_dt,
-            }"/>
-            <i class="fa-stack-1x o-livechat-LivechatStatusLabel-icon" t-attf-class="{{ livechatThread.livechat_end_dt ? 'fa fa-flag-checkered' : store.livechatStatusButtons.find(btn => btn.status === livechatThread.livechat_status)?.icon }}" t-att-class="{
-                'o-warning': livechatThread.livechat_status === 'waiting',
-                'o-info': livechatThread.livechat_status === 'need_help',
+            <t t-log="env"/>
+            <i t-if="!env.inDiscussSidebar" class="o-livechat-LivechatStatusLabel-icon fa-fw" t-attf-class="{{ livechatThread.livechat_end_dt ? 'fa fa-flag-checkered' : store.livechatStatusButtons.find(btn => btn.status === livechatThread.livechat_status)?.icon }}" t-att-class="{
                 'o-white': livechatThread.livechat_end_dt,
             }"/>
         </span>
     </t>
 
     <t t-name="im_livechat.LivechatStatusSelection">
-        <div class="o-livechat-LivechatStatusSelection btn-group d-flex flex-column o-rounded-bubble o-outline-secondary shadow-sm" t-ref="livechat-status-selection">
+        <div class="o-livechat-LivechatStatusSelection list-group" t-ref="livechat-status-selection">
```

Comment:
In dark theme the "In progress" and "Looking for help" items use almost the same colors, which doesn't help knowing which item is selected at a glance other than order.
<img width="737" height="214" alt="Screenshot 2025-08-29 at 15 54 03" src="https://github.com/user-attachments/assets/bf351930-0f69-413e-a77b-57d4f744720b" />

Different colors help scanning to know for sure which item is selected. Only having different order requires more effort to know for sure which item is selected, so this is less efficient. "Waiting for customer" is good with its own color:

<img width="291" height="180" alt="Screenshot 2025-08-29 at 15 54 16" src="https://github.com/user-attachments/assets/932a542a-183d-462f-bce8-cf569b2b8dbf" />

Reply:
The similarity between `action` and `info` color is a known issue, especially when these colors are used for borders or small amount of text. In will be addresses in a proper task and at framework level.

For the time being, I've opted for a different color combination using another bootstrap default value